dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -1698,7 +1699,11 @@ public void TopNavigationWithAccessKeysTest()
                 InvokeNavigationViewAccessKeyAndVerifyKeyTipPlacement("TopNavOverflowButton");
 
                 Log.Comment("Verify overflow menu is opened");
-                Verify.IsTrue(GetTopNavigationItems(TopNavPosition.Overflow).Count > 0);
+                // Flyout doesn't seem raise any UIA WindowOpened/MenuOpened events so just check a few times for the menu to
```

Comment:
> Flyout doesn't seem raise any UIA WindowOpened/MenuOpened events

Should it?

Reply:
Probably, but I think that's in the framework. I'll follow up.